ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -56,9 +58,59 @@ message GetActiveContractsResponse {
 
   // The workflow that created the contracts.
   // Must be a valid LedgerString (as described in ``value.proto``).
+  // Optional, In the Ledger API 3.0 this field will be removed.
   string workflow_id = 2;
 
   // The list of contracts that were introduced by the workflow with ``workflow_id`` at the offset.
   // Must be a valid LedgerString (as described in ``value.proto``).
+  // In the Ledger API 3.0 this field will be removed.
+  // This is empty for multi-domain API.
   repeated CreatedEvent active_contracts = 3;
+
+  // Non-empty for multi-domain API.
```

Comment:
what ordering do you expect on the events? I assume for incomplete transfers it's just the ordered by the participant offset of the corresponding transfer out/transfer in. What about the active contract events? The offset of the activation event, i.e., the transfer in or create?

Reply:
Well currently the order is not defined, and it is strictly increasing in terms of offset/event-seq-id.
It seems logical now to have the same order, to pave the way to resumable ACS.
WDYT? Do you need order?

Reply:
as discussed we do not commit to order, eventually with high prob it will ordered by offset/seqid